base/Angle.hpp

```diff
@@ -106,6 +107,20 @@ class Angle
         return result;
     }
 
+    /** Use this method to get the minimum acceptable angle
```

Comment:
I agreed with you, @jmachowinski and @doudou. 

I followed the second @doudou's suggestion: "Returns the minimum angle possible after normalization".

Code is already updated.